cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,76 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const Password = 'P@ssword123';
+  const UserName = 'SnTest';
+  const BookName = 'Speaking JavaScript';
+  const BookDescription =
+  'Like it or not, JavaScript is everywhere these days-from';
+  beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it('Should provide an ability to log in with valid credentials', () => {
+    cy.get('#userName')
+      .type(UserName);
+
+    cy.get('#password')
+      .type(Password);
+
+    cy.contains('button', 'Login')
+      .click();
+
+    cy.get('#userName-value')
+      .should('contain', 'SnTest');
+
+    cy.url().should('eq', 'https://demoqa.com/profile');
+  });
+
+  it('Should provide an ability to add the book to collection', () => {
+    cy.login(UserName, Password);
```

Comment:
The custom command cy.login() is supposed to log in with API but it seems to be used without any implementation of this functionality.

Reply:
Done